docs/contributing/developing.rst

```diff
@@ -179,6 +185,28 @@ IE             Desktop        10 and below
 Safari         Windows        All
 =============  =============  =============
 
+Accessibility targets
+---------------------
+
+We want to make Wagtail accessible for users of a wide variety of assistive technologies. The specific standard we aim for is `WCAG2.1 <https://www.w3.org/TR/WCAG21/>`_, AA level. Wagtail’s administration user interface isn’t accessible at the moment (see `issue #4199 <https://github.com/wagtail/wagtail/issues/4199>`_), but here are specific assistive technologies we aim to test for, and ultimately support:
+
+=============  ====================
+Type           Assistive technology
+=============  ====================
+Screen reader  `NVDA <https://www.nvaccess.org/download/>`_ on Windows with Firefox ESR
+Screen reader  `VoiceOver <https://support.apple.com/en-gb/guide/voiceover-guide/welcome/web>`_ on macOS with Safari
+Magnification  `Windows Magnifier <https://support.microsoft.com/en-gb/help/11542/windows-use-magnifier>`_
+Magnification  macOS Zoom
+Voice control  Windows Speech Recognition
+Voice control  macOS Dictation
+=============  ====================
+
+We aim for Wagtail to work in those environments. Our development standards ensure that the site is usable with other assistive technologies. In practice, testing with assistive technology can be a daunting task that requires specialised training – here are tools we rely on to help identify accessibility issues, to use during development and code reviews:
+
+* `react-axe <https://github.com/dequelabs/react-axe>`_ integrated directly in our build tools, to identify actionable issues.
```

Comment:
Might be worth adding that messages appear in the developer console - I expected to see some kind of UI at first before switching to the console tab.